src/ViewItem.js

```diff
@@ -240,9 +247,33 @@ class ViewItem extends React.Component {
     this.props.updateLocation({ layer: 'copyItem' });
   }
 
+  handleConfirm = (item, requestRecords) => {
+    const newItem = _.cloneDeep(item);
+    _.set(newItem, ['status', 'name'], 'Missing');
+
+    const newRequestRecord = _.cloneDeep(requestRecords);
```

Comment:
@aditya-matukumalli can we just clone `requestRecords[0]` here?

Reply:
@mkuklis Good catch. Will make the change